src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
+    # We are going to prepend this to compilation tool invocations, so we need to take
+    # the specified host triple and at a '-'
+    HOST=`expr "X$1" : '[^=]*=\(.*\)'`-;
+    shift ;;
+  --fst-version=*)
```

Comment:
My bad, I haven't thought to check the commit messages. @danpovey I am ok with this change. It seems like a valid use case and it won't break existing stuff.